flask_socketio/__init__.py

```diff
@@ -70,6 +70,15 @@ def process_event(self, packet):
                 app = self.request
                 return self.socketio._dispatch_message(app, self, message, args)
 
+            def get_room(self, room):
+                if not self.socketio.rooms.get(self.ns_name, False):
+                    return []
+                return list(self.socketio.rooms[self.ns_name].get(room, []))
```

Comment:
So this was one of my concerns. This function is returning a private object maintained by gevent-socketio, a project that has not been maintained in a long time.

I'm hoping to replace gevent-socketio in the future, so returning this private structure makes my life harder, as I will then be forced to continue using the gevent-socketio structures, or else break current users when I switch to the new thing.

My preference would be to abstract the application from these internal structures. Here you could return a list of client IDs for example, and then add new option to the emit and send methods to take this ID to direct a message to a specific user.